client/src/templates/Challenges/projects/backend/Show.js

```diff
@@ -175,15 +161,12 @@ export class BackEnd extends Component {
         challengeMeta: { introPath, nextChallengePath, prevChallengePath }
       },
       tests,
-      isSignedIn,
       executeChallenge,
-      updateProjectFormValues
+      updateSolutionFormValues
```

Comment:
```suggestion
      updateSolutionFormValues,
      updateBackendFormValues
```
This is needed for challenges.